waves-ext/src/main/scala/com/wavesplatform/dex/grpc/integration/DEXExtension.scala

```diff
@@ -31,14 +31,15 @@ class DEXExtension(context: ExtensionContext) extends Extension with ScorexLoggi
 
     val host: String = context.settings.config.as[String]("waves.dex.grpc.integration.host")
     val port: Int = context.settings.config.as[Int]("waves.dex.grpc.integration.port")
+    val permitKeepAliveTime: FiniteDuration = context.settings.config.as[FiniteDuration]("waves.blockchain-updates.min-keep-alive")
 
     val bindAddress = new InetSocketAddress(host, port)
     apiService = new WavesBlockchainApiGrpcService(context)
 
     val builder = NettyServerBuilder
       .forAddress(bindAddress)
       .permitKeepAliveWithoutCalls(true)
-      .permitKeepAliveTime(500, TimeUnit.MILLISECONDS)
+      .permitKeepAliveTime(permitKeepAliveTime.length, permitKeepAliveTime.unit)
```

Comment:
I think we don't need this, because it is our extension and it doesn't depend on Blockchain Updates Extension. Thus we could have own settings